CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # vscode-powershell Release History
 
+## v2.0.0-preview.3
+### Wednesday, April 10, 2019
+#### [vscode-powershell](https://github.com/powershell/vscode-powershell)
+
+- [vscode-PowerShell #1852](https://github.com/PowerShell/vscode-PowerShell/pull/1852) -
+  Turn `powershell.codeformatting.useCorrectCasing` setting off by default until PSSA issues are fixed (Thanks @bergmeister!)
+- [vscode-PowerShell #1838](https://github.com/PowerShell/vscode-PowerShell/pull/1838) -
+  Set PSReadLine featureFlag default to null so that it can be resolved by settings
+- [vscode-PowerShell #1837](https://github.com/PowerShell/vscode-PowerShell/pull/1837) -
+  Do not use -EncodedCommand on Windows
+- [vscode-PowerShell #1825](https://github.com/PowerShell/vscode-PowerShell/pull/1825) -
+  Switch to current lowercase names for powershell and mdlint recommended extensions (Thanks @rkeithhill!)
```

Comment:
What about the WhitespaceAroundPipe and WhitespaceBeforeBrace settings? I feel the settings are so new we could risk the breaking change of having them lower-cased

Reply:
I think we should do that too